easybuild/easyconfigs/h/HDF5/HDF5-1.14.5-NVHPC-24.9-CUDA-12.6.0-serial.eb

```diff
@@ -0,0 +1,23 @@
+name = 'HDF5'
+# Note: Odd minor releases are only RCs and should not be used.
+version = '1.14.5'
+versionsuffix = "-serial"
+
+homepage = 'https://portal.hdfgroup.org/display/support'
+description = """HDF5 is a data model, library, and file format for storing and managing data.
+ It supports an unlimited variety of datatypes, and is designed for flexible
+ and efficient I/O and for high volume and complex data."""
+
+toolchain = {'name': 'NVHPC', 'version': '24.9-CUDA-12.6.0'}
```

Comment:
i think it's better to add `-CUDA-12.6.0` to the versionsuffix instead of the version

Reply:
Done. I hope that does not break anything.

Reply:
That seemed to have broken something. :-(